packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright 2015 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'dart:math';
```

Comment:
Flutter standard is to import math like so:

import 'dart:math' as math;

(Just because some of the values in math have very short names, like pi and e, that might conflict.)

Reply:
Done!